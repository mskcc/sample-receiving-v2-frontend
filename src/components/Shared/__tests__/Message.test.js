import React, { Component } from 'react'
import renderer from 'react-test-renderer'
import TestMessage from '../Message'

const Message = props => (

    <TestMessage />
  
)

describe('Render Message', () => {
  it('render Message correctly', () => {
    const MessageComponent = renderer.create(<Message />).toJSON()
    expect(MessageComponent).toMatchSnapshot()
  })
})
