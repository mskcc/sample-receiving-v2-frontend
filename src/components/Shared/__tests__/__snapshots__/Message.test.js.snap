// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Message render Message correctly 1`] = `
<div>
  <div
    className="MuiPaper-root-2 MuiPaper-elevation1-5 MuiPaper-rounded-3 Message-container-1"
  >
    <h3
      className="MuiTypography-root-29 MuiTypography-h5-45"
    >
      Error
       
    </h3>
    <p
      className="MuiTypography-root-29 MuiTypography-body1-38"
    />
  </div>
</div>
`;
